app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
+
+    def __init__(
+            self,
+            name: str
+    ) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
You shouldn't redefine the skills attribute.
You should extend it in the __init__ method.

From the checklist:
Use the extend() method or += operator to add multiple values to a single list.